### DISCUSSION TITLE
Question: How do I get the VSCode extension to use rome.json

### DISCUSSION THREAD

**Discussion by oriongonza** [rome/tools#3492] [Q&A]
It ignores it, but the CLI works properly. How do I configure it?

**Comment by ematipico** [comment]
As for now, you would need to open your project as the root of your workspace, and `rome.json` to stay at the same level of the `package.json`.

<img width="312" alt="Screenshot 2022-10-26 at 10 34 00" src="https://user-images.githubusercontent.com/602478/197991441-37fff682-8784-491c-b972-bdcf33537f85.png">

**Reply by oriongonza** [reply]
I have it like that though
![image](https://user-images.githubusercontent.com/47633543/197997651-aa03fbd0-c688-4543-b805-7898500bd3b6.png)

**Reply by ematipico** [reply]
Feel free to open an issue, or open a thread on Discord. We might need more information around the content of the `rome.json` and if the underlying daemon is running the correct client.

**Reply by oriongonza** [reply]
> if the underlying daemon is running the correct client.

I don't know how to check that. 

I've opened an issue.
